Lethal-Lemurs/Bangazon-Workforce-Mgt
# Added all other pug files as well as the ability to route through pug...

to individual urls in prep for viewing individual details. Nav should route you to /employees and /departments now. The create buttons don't do anything yet. Also we need functions for the individual details to work too.

To test:

1. npm install if you haven't already
2. run npm start
3. then go to localhost 8080 in your browser of choice
4. use the nav bar to see if the url changes to /employees and /departments, the page should display errors that's ok.
5. then go to /computers after that click a computer and see if the url has /computers/1 or 2 or 3
6. Check to see that forms are serviceable.
 
Files changed: 
modified:   views/computers.pug
modified:   views/nav.pug
Files added:
        views/computer-details.pug
        views/create-department.pug
        views/create-employee.pug
        views/department-details.pug
        views/departments.pug
        views/employee-details.pug
        views/employees.pug
